### PR TITLE
add janet version

### PR DESCRIPTION
**Pull request by add1ed**

### CODE REVIEW

File: janet/test/gilded-rose.janet [hunk 1]
```diff
@@ -0,0 +1,10 @@
+(import ../gilded-rose/main)
+(use judge)
+
+(do
+  (def items 
+    [@{ :name "foo" :quality 3 :sell-in 5}
```
Comment (emilybache) [inline janet/test/gilded-rose.janet:6]: This doesn't quite match the starting unit test of the other languages. Could you change the array to only contain "foo" and give it quality 0 and sell-in 0 ?

File: janet/test/gilded-rose.janet [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+     @{ :name "bar" :quality 6 :sell-in 6}])
+  (main/update-quality items)
+  (def found (find |(= ($ :quality) 2) items))
```
Comment (emilybache) [inline janet/test/gilded-rose.janet:9]: I think this is an assertion that is also an addition compared to the other languages? I wasn't sure what this line does.

**Comment by emilybache** [comment]
Thankyou! This looks very promising. The unit test is just a little too sophisticated! If you want to help people to use the test framework, perhaps you could also add a README file and link to the documentation?

File: janet/test/gilded-rose.janet (hunk 1 above)
Comment (add1ed) [reply janet/test/gilded-rose.janet]: Sorry! I have overlooked that all the other implementations had exactly the same unit test. I've altered the janet test to look more like everyone else's.

**Comment by add1ed** [comment]
I have added a commit with the updates you asked for 👍 
You may find the [testing library](https://github.com/ianthehenry/judge) being used of interest - it is focused on inline snapshot testing and has some similarities to texttest.

**Comment by emilybache** [comment]
Thankyou, happy to hear about another snapshot testing tool too

**Pull request merged**